packages/rancher-sriov/templates/crd-template/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+version: v1.0.0
```

Comment:
The upstream sriov project we are deploying is using v1.0.0 that's why I left it here. I guess it does not matter but, Is it mandatory to remove the "v"?

Reply:
I don't think v1.0.0 [is valid semver](https://semver.org/#spec-item-2) but I'm not sure if this would cause an issue with Helm today. to be safe, I think we should remove it.